examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -1634,6 +1634,20 @@ THREE.LegacyGLTFLoader = ( function () {
 									geometry.addAttribute( 'skinIndex', bufferAttribute );
 									break;
 
+								default:
+									var material = json.materials[ primitive.material ];
+									var parameters = json.techniques[ material.technique ].parameters;
```

Comment:
The primitive may not have a material, the material may not have a technique, and the technique may not have parameters. In any of those cases we should `break` rather than throwing an error, I think.